packages/react-components/src/lib/utils/networkConfig.ts

```diff
@@ -0,0 +1,22 @@
+export type NetworkNotice = {
```

Comment:
Let's export these from packages/react-components/src/lib/utils/index.ts as well so they're accessible to consumers and in the typedocs